src/server/db_slice.h

```diff
@@ -62,6 +62,42 @@ class DbSlice {
   void operator=(const DbSlice&) = delete;
 
  public:
+  class AutoPostUpdate {
+   public:
+    AutoPostUpdate();
+    AutoPostUpdate(AutoPostUpdate&& o);
```

Comment:
`AutoPostUpdate(AutoPostUpdate&& o) = default` does not work here?

Reply:
no, because we need to set `o.action` to `kDoNothing`, otherwise both will run...

Reply:
Ah, yes, it's literally my comment above 🤣